install-dev/data/xml/tab.xml

```diff
@@ -215,32 +215,6 @@
                         <wording>Stock Management</wording>
                         <wording_domain>Admin.Navigation.Menu</wording_domain>
                     </tab>
-                    <tab id="Stock_Movement" id_parent="Stock_Management" active="1" enabled="1" icon="">
```

Comment:
indentation is weird for this file. Maybe we should introduce a children node to avoid this "logical" tabs

Not related to this PR btw